packages/scripts/config/webpack.config.js

```diff
@@ -58,6 +58,21 @@ const externals = [
 const isProduction = process.env.NODE_ENV === 'production';
 const mode = isProduction ? 'production' : 'development';
 
+const babelLoader = hasBabelConfig() ? {
+	test: /\.js$/,
+	exclude: /node_modules/,
+	use: require.resolve( 'babel-loader' ),
+} : {
+	test: /\.js$/,
+	exclude: /node_modules/,
+	use: {
+		loader: require.resolve( 'babel-loader' ),
+		options: {
```

Comment:
The only difference here are `options` so maybe it would be possible to consolidate this code in a way where only `options` are provided conditionally.

Reply:
addressed at f7c1a04